test_requirements.txt

```diff
@@ -1 +1,2 @@
-tensorflow>=2.4.1
\ No newline at end of file
+tensorflow>=2.4.1
+h5py
```

Comment:
You'll want to add this to requirements.txt if you do a plain "import h5py" (without the try: except ImportError I suggested above) in the main package, because then paltas won't run if it is not installed.